src/uistate.h

```diff
@@ -150,6 +151,10 @@ class uistatedata
 
         std::map<ammotype, itype_id> lastreload; // id of ammo last used when reloading ammotype
 
+        // wide and tall display
+        bool wide_display = false;
+        bool full_height_display = false;
```

Comment:
And here, on the contrary, it should have verbose name because uistate is used by the entire codebase, and there are dozens of UIs in game, so it's unclear which UI this `wide_display` refers to. Something like:
```suggestion
        // Wide and tall display for messages window
        bool msg_window_wide_display = false;
        bool msg_window_full_height_display = false;
```